src/components/NavBar/components/ImageLocaleWrapper/index.js

```diff
@@ -25,12 +25,14 @@ export default class ImageLocaleWrapper extends Component {
   }
 
   render() {
-    const { intl, src, alt, className } = this.props;
+    const { intl, src, alt, className, onClick } = this.props;
     return (
+      // eslint-disable-next-line
```

Comment:
what needs to be disabled?

Reply:
will have this warning, Non-interactive elements should not be assigned mouse or keyboard event listeners